pkg/etc/wavefront/wavefront-proxy/log4j2.xml.default

```diff
@@ -88,6 +88,12 @@
             <AppenderRef ref="ValidPointsFile"/>
             -->
         </AsyncLogger>
+        <!-- Uncomment AppenderRef and set level="ALL" to log Jaeger data to a file -->
+        <AsyncLogger name="JaegerDataLogger" level="DEBUG" additivity="false">
+            <!--
+            <AppenderRef ref="JaegerDataFile"/>
```

Comment:
If we're adding a reference to JaegerDataFile, should we define it as well (and comment out)?